tests/test_optic.py

```diff
@@ -114,6 +115,22 @@ def test_shift():
         assert telescope == shifted
 
 
+@timer
+def test_rotXYZ_parsing():
```

Comment:
Need to add this to the `if __name__ == '__main__':` block.

Reply:
Done.